db/version_builder.cc

```diff
@@ -81,8 +81,13 @@ class VersionBuilder::Rep {
   };
 
   struct LevelState {
-    std::unordered_set<uint64_t> deleted_files;
-    // Map from file number to file meta data.
+    // Files in base version that should be deleted.
+    std::unordered_set<uint64_t> deleted_base_files;
```

Comment:
revert to `deleted_files`

Reply:
But they are of different meanings, previously all files including newly added ones can be deleted by this field.

Reply:
ok